README.md

```diff
@@ -26,11 +27,19 @@ yarn
 yarn build
 ```
 
+#### Configure
+
+You need WalletConnect Cloud `projectId` to run Hyperlane Warp Route UI successfully. 
+Sign up to [WalletConnect Cloud](https://cloud.walletconnect.com/), create 
+new project with AppKit and Next.js and copy `projectId` from there.
+
 ### Run
 
+You need to set up environment variable `NEXT_PUBLIC_WALLET_CONNECT_ID` to `projectId` you have got from WalletConnect
+Cloud.
```

Comment:
```suggestion

To use this `projectId`, either set the `NEXT_PUBLIC_WALLET_CONNECT_ID` environment variable, or update the `walletConnectProjectId` in the [constants config](https://github.com/hyperlane-xyz/hyperlane-warp-ui-template/blob/c9637f9e2755cf4243ec45e80265ea55c40d4f5a/src/consts/config.ts#L7).

### Run
```

Reply:
also +1 to rossy's `.env.local` comment

Reply:
Changing `config.ts` feels too tactical. I would not recommend it to users. I'll add Rossy's suggestion.